### PR TITLE
Issue #1161 by JrgenGNielsen: Revert check on empty()

### PR DESCRIPTION
**Pull request by JrgenGNielsen**

### CODE REVIEW

File: Bpi/Sdk/Bpi.php [hunk 1]
```diff
@@ -94,7 +94,7 @@ public function push(array $data)
         $nodes->firstItem('name', 'node')
             ->template('push')
             ->eachField(function ($field) use ($data) {
-                if (!empty($data[(string)$field]))
+                if (!isset($data[(string)$field])) // nb: check on empty() produce errors.
```
Comment (guddo) [inline Bpi/Sdk/Bpi.php]: https://www.drupal.org/coding-standards/docs#inline

Comment (kasperg) [inline Bpi/Sdk/Bpi.php]: I agree with @guddo here. If possible please put comments on a separate line.

In general I do not think the BPI client which should not be tied to Drupal specifically must follow Drupal coding standards. If we were to follow Drupal coding standards [the same line would violate control structure standards](https://www.drupal.org/coding-standards#controlstruct).

Comment (kasperg) [inline Bpi/Sdk/Bpi.php]: In general I would prefer a comment on _why_ we use `isset` and not `empty`. I am guessing that these templates support empty values but require callers to define them if so.

**Comment by kasperg** [comment]
Reviewed and approved.

**Pull request merged**